### ISSUE TITLE
Documentation for each param in vpatient_params.csv

### ISSUE THREAD

**Issue by PorkShoulderHolder** [jxx123/simglucose#27]
Not sure how to interpret each parameter in this file.  It also looks like all of the `x0_` parameters are not used, so do those have any purpose?  

Ideally there would be a document which provides a sentence description for each one.

Thanks.

**Comment by manueich** [comment]
On the same topic, I was wondering if you could cite a source for the parameter values. 
As far as I am aware, these paramter values are only available in the context of the commercially available software package of The Epsilon Group.

Thanks,

**Comment by PorkShoulderHolder** [comment]
@manueich this may be helpful: https://github.com/jxx123/simglucose/issues/26, but I'm going to leave the issue open b/c there really should be documentation in the repo.

**Comment by jdmedinatobon** [comment]
I also agree there should be more documentation in the repo. I've been trying to decipher in which units each variable and parameter are. Has anyone been able to find this information? Perhaps by contacting the The Epsilon Group or similar?

**Comment by aespogom** [comment]
Hi there!
Has anyone else found information about this?
I would like to try the simulator with different parameters but the naming is not clear to me in many fields.
Thanks!

**Comment by edger-asiimwe** [comment]
Hi there @aespogom 

You can find information on some of the parameters in this [paper](https://ieeexplore.ieee.org/abstract/document/4303268?casa_token=zJvUmbOB678AAAAA:KyEEJOqFkYr0coTKi5Bn2_bjn4ouAFmdfh5kIncdrd34nuJiOTIpgIgAM3UKuIKY7XANWWp84eTkMg). The paper does not cover the entire parameters as in the simulator, and/or patient parameters, but it identifies the parameters for the different model compartments. 

Hope it helps

**Comment by RobotPsychologist** [comment]
@PorkShoulderHolder  @jxx123 with #88 I believe we can now close this issue?
